### PR TITLE
Migrated breaking changes for use with Hotchocolate 14

### PR DESCRIPTION
**Pull request by d-lundy**
QueryResult is now OperationResult

### CODE REVIEW

File: src/AppAny.HotChocolate.FluentValidation.csproj [hunk 1]
```diff
@@ -33,8 +33,8 @@
 <!--  </ItemGroup>-->
 
   <ItemGroup Label="Packages">
-    <PackageReference Include="FluentValidation" Version="11.9.0" />
-    <PackageReference Include="HotChocolate.Execution" Version="13.8.1" />
+    <PackageReference Include="FluentValidation" Version="11.10.0" />
+    <PackageReference Include="HotChocolate.Execution" Version="14.0.0" />
```
Comment (m4tchl0ck) [inline src/AppAny.HotChocolate.FluentValidation.csproj]: Consider changing the version to newest
```suggestion
    <PackageReference Include="HotChocolate.Execution" Version="14.1.0" />
```

**Review by m4tchl0ck — CHANGES_REQUESTED** [review]

File: tests/AppAny.HotChocolate.FluentValidation.Tests/SkipValidation.cs [hunk 2]
```diff
@@ -29,14 +29,14 @@ public async Task FieldSkipValidation()
           services.AddTransient<IValidator<TestPersonInput>, NotEmptyNameValidator>();
         });
 
-      var result = Assert.IsType<QueryResult>(
+      var result = Assert.IsType<OperationResult>(
         await executor.ExecuteAsync(TestSetup.Mutations.WithEmptyName));
 
-      result.AssertSuceessResult();
+      result.AssertSuccessResult();
```
Comment (m4tchl0ck) [inline tests/AppAny.HotChocolate.FluentValidation.Tests/SkipValidation.cs:35]: Would you happen to know why the row changed?
I don't see any difference, but maybe it's encoding or endline

Comment (d-lundy) [reply tests/AppAny.HotChocolate.FluentValidation.Tests/SkipValidation.cs:35]: Suceess => Success for spelling

File: src/AppAny.HotChocolate.FluentValidation.csproj (hunk 1 above)
Comment (d-lundy) [reply src/AppAny.HotChocolate.FluentValidation.csproj]: Updated to 14.1.0

**Review by vvoldsund — APPROVED** [review]

**Review by m4tchl0ck — APPROVED** [review]

**Comment by mormegill** [comment]
Awesome work! I would love to see it accepted soon as it's the only thing blocking us from a HC14 migration

**Comment by eduardogoncalves** [comment]
@sergeyshaykhullin can you merge it?

**Comment by sergeyshaykhullin** [comment]
@eduardogoncalves Released

https://www.nuget.org/packages/AppAny.HotChocolate.FluentValidation#versions-body-tab

**Comment by eduardogoncalves** [comment]
Thank you @d-lundy, @m4tchl0ck and @sergeyshaykhullin  for your valuable contribution to the project! Your effort and dedication help strengthen the community!

**Pull request merged**